dhis-2/dhis-api/src/main/java/org/hisp/dhis/interpretation/Interpretation.java

```diff
@@ -407,7 +462,7 @@ public void setPeriod( Period period )
 
     @JsonProperty
     @JsonSerialize( as = BaseIdentifiableObject.class )
-    @JacksonXmlProperty( namespace = DxfNamespaces.DXF_2_0 )
```

Comment:
Just a minor tip for next time - it can be a good idea to do the general clean-up of code in a separate PR and get that merged. Then the PR for the new feature becomes a bit more concise and readable.

Reply:
Okay, forgot that. But will remember next time :+1: